core/utilities/Logger.h

```diff
@@ -26,30 +28,38 @@ class Log;
 
 enum class LogLevel
 {
-    TRACE = 0,
-    DEBUG,
-    INFO,
+    INFO = 0,
     WARN,
-    ERROR
+    ERROR,
+    DEBUG,
+    TRACE,
+    OFF
 };
 
 /**
  * @brief Converts a log level from string.
  * @param level log level as string
- * @return coresponding log level (supported: ERROR, WARN, INFO, DEBUG and
- * TRACE. All other will be considered as ERROR)
+ * @return corresponding log level (supported: ERROR, WARN, INFO, DEBUG, TRACE and
+ * OFF. All other will be considered as ERROR)
  */
 wolkabout::LogLevel from_string(std::string level);
 
 /**
  * @brief The Logger class
  *
  * Abstract class that should be extended by specific logger. Logger is accessed
- * as a signeton. But this single instance must be set via #setInstance.
+ * as a singleton. But this single instance must be set via #setInstance.
  */
 class Logger
 {
 public:
+    enum class Type
```

Comment:
at "enum class LogLevel" you start eum from 0, here from 1. Sync it.

Reply:
```
    enum class Type
    {
        CONSOLE = 1,
        FILE = 2,
        BUFFER = 4
    };
```
Think the 1 is to allow for having multiple types of logging enabled